synapse/replication/tcp/protocol.py

```diff
@@ -578,42 +537,6 @@ def connectionMade(self):
         # Once we've connected subscribe to the necessary streams
         self.replicate()
 
-        # Tell the server if we have any users currently syncing (should only
-        # happen on synchrotrons)
-        currently_syncing = self.handler.get_currently_syncing_users()
-        now = self.clock.time_msec()
-        for user_id in currently_syncing:
-            self.send_command(UserSyncCommand(self.instance_id, user_id, True, now))
```

Comment:
has this got lost?

Reply:
Err, yes

Reply:
This got added back as part of rebase, though should be moved

Reply:
(moved)